pkg/middleware/request_metrics.go

```diff
@@ -23,7 +24,17 @@ func init() {
 		},
 	)
 
-	prometheus.MustRegister(httpRequestsInFlight)
+	httpRequestDurationHistogram = prometheus.NewHistogramVec(
```

Comment:
Is there any reason why httpRequestDurationHistogram is initialized only after declaration? Why not initialize at the same time of declaring the variable?

Reply:
I have to admit that that's the pattern I've seen in other places. But that doesn't mean it makes sense. I'll double-check.

Reply:
Thanks :) I think it's safer to initialize on declaration, and this should be the pattern unless there's a reason not to.